posts/coalesce2022-a-recap.md

```diff
@@ -0,0 +1,238 @@
+> - Which are the most frequently accessed data assets?
+> - Who's accessing sensitive data and how often?
+
+14. Data Deletion Requests
+> Problem: How do you handle consumer requests to delete data?
+> 
+> Solution: Atlan allows you to track the data across your ecosystem and make sure it is actually deleted.
+
+15. Optimize data stack spending?
+> Problem: How can you identify the following:
+> - Which queries are taking too long to run?
+> - Which tables can we deprecate?
+> - How long are jobs waiting to execute?
+> - What tables are used by BI tools?
+> - What are the most/least queried tables?
+> 
+> Solution: All of this should be generating metadata in your system. You can then leverage that metadata to better optimize usage.
+
+16. Active spend optimization
+> Problem: How do I handle peak usage at 10AM of my data warehouse?
+> 
+> Solution: You can leverage metadata to auto-scale your compute clusters at 10 AM and make sure your pipeline is updated at 9:45. You can then auto-scale everything down when users log-off.
+
+17. Active performance optimization
+> Problem: How do I ensure the best user experience for data consumers?
+> 
+> Solution: If you have metadata across your ecosystem and metadata driven actions, you can orchestrate your data platform to create the best user experience for your consumers.
+> 
+> ![Active Spend and Performance Optimizations](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/11/coalesce_20_metadata_active_spend_and_performance_optimizations.png) 
+
+18. Write better SQL queries with context
+> Problem: How can I do my job better?
+> 
+> Solution: Make the metadata context available to tools used on a daily basis. Integrate your metadata into your IDE so you can see recent queries run on a table, is it a popular table, who owns it, are there issues with the table, what a metric means, the column description and many other things.
+
+19. Make decisions self-service fo ryour business
+> Problem: How do you ensure the business is making data-driven decisions in a self-service manner?
+> - Can I trust this data?
+> - What does the metric mean?
+> - Who can I reach out to about the data/metric?
+> - Can I run a quick filter/pivot on this metric?
+> 
+> Solution: You can integrate Atlan into your BI tool to show all of the metric context from the dbt Semantic layer right in the dashboard. So you have all of the data context to make decisions
+
+20. Enable data teams that can change the world
+> Problem: How do you unify and enable a diverse team to facilitate data to create change?
+> 
+> Solution: Embrace the diversity in a team. It is a data team's biggest strenght and weakness. Atlan's metadata is what can help a team gel with each other and do great things.
+
+[Watch Prukalpa's Full Talk here](https://www.youtube.com/watch?v=fxFxfKCZrhY&list=PL0QYlrC86xQlj9UDGiEwhXQuSjuSyPJHl&index=78)
+[For more information on all that Atlan can offer](https://atlan.com/)
+
```

Comment:
This recap of the 20 questions in the midst of the other recap summaries is a bit jarring.  I think it's probably fine to leave all 20 points, but consider hitting the top 3 or 5 instead.  Including all 20 points turns this blog post into a 10+ minute read.